types/k6/global.d.ts

```diff
@@ -80,4 +80,622 @@ declare global {
          */
         resolve(specifier: string): string;
     }
+
+    /**
+     * The `crypto` provides the access to the k6's WebCrypto API implementation.
+     */
+    const crypto: Crypto;
+
+    interface CryptoKey {
+        /**
+         * The type of key the object represents.
```

Comment:
Not certain this is really needed as it is a well known and standartized global - if anything I think this might actually create a problem in the future. 

For me at least this works with no problems and no additional changes even without those changes 

![image](https://github.com/user-attachments/assets/41b3f987-f7a5-4f7c-8ea5-a31ab205a5cc)

Reply:
Do you mean we don't need to define to make definitions for `crypto` at all? 

>it is a well known and standartized global

But we don't fully match global, it's still an implementation...

Reply:
> Do you mean we don't need to define to make definitions for crypto at all?

Yes

> But we don't fully match global, it's still an implementation...

1. this is hopefully going to change quick enough that it won't matter
2. Again I am not certain that this definition here helps if the other global one is also installed, and I expect the otehr is more of "default" one so it likely will be used either way.

Reply:
I also really do not want us to have to both implement and then write/copy-paste type deifntions for eveyr standard(ish) API we add to k6

Reply:
I mean -we still should depracte teh experimental one : P

Reply:
Yeah, but it's easier and more straight forward to cherry pick the deprecation commit to release branch, I've openned this PR because wasn't sure about the approach